### PR TITLE
Support for Insteon Hub Version 1

### PR DESCRIPTION
**Pull request by teharris1**
**Description:**
Added configuration settings for the older Insteon Hub.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16472

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ source/.jekyll-metadata
 /.vs/slnx.sqlite-journal
 /.vs/VSWorkspaceState.json
 .vscode
+*.suo
```
Comment (MartinHjelmare) [inline .gitignore:23]: Is this intended?

Comment (teharris1) [reply .gitignore:23]: Yes but OK to take it out. It is for Visual Studio Pro.  VS Code is .vscode but pro is .suo.

Comment (MartinHjelmare) [reply .gitignore:23]: Keep it then.

File: source/_components/insteon.markdown [hunk 2]
```diff
@@ -234,9 +255,8 @@ events. The following events are available:
     the values are a to d. For an 8 button remote the values are a to g. For
     a one button remote this field is not used.
 
-This allows the mini-remotes to be configured as 
-
-Here is an example of how to use these events for automations:
+This allows the mini-remotes to be configured as triggers for automatoins. Here
```
Comment (MartinHjelmare) [inline source/_components/insteon.markdown]: automations

**Comment by frenck** [comment]
Tagged 0.78, since the parent PR has been tagged as well.

**Comment by frenck** [comment]
I'm picking this one up. There are quite a lot of small grammar and spelling error in here.

**Comment by teharris1** [comment]
@frenck Thanks for helping out. I did put this through a quick spelling and grammar check and made the changes found. Please let me know if you find others. Thanks again.

**Comment by frenck** [comment]
@teharris1 Thanks! I was already halfway through... 😠  😉 

I'll reset and check from scratch 👍

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for the PR @teharris1! 👍

**Pull request merged**